### ISSUE TITLE
[JOSS REVIEW] first example fails

### ISSUE THREAD

**Issue by thelfer** [NGSolve/ngsPETSc#56]
The following example, taken from https://ngspetsc.readthedocs.io/en/latest/PETScKSP/poisson.py.html, fails as follows:

~~~
$ python3 poisson.py 
optfile ./ng.opt does not exist - using default values
togl-version : 2
loading ngsolve library
NGSolve-6.2.2405-28-g63cfc18ca
Using Lapack
Including sparse direct solver UMFPACK
Running parallel using 32 thread(s)
Traceback (most recent call last):
  File "/home/th202608/codes/ngsPETSc/tests/Poisson/poisson.py", line 7, in <module>
    ngmesh = unit_square.GenerateMesh(maxh=0.1)
NameError: name 'unit_square' is not defined
~~~

**Comment by UZerbinati** [comment]
This example is running for me on a fresh install and on the docker image.
Could it be that something went wrong with your ngsolve install?

**Comment by thelfer** [comment]
I just followed your tutorial steps by steps. `ngsolve` did get installed, so I expect that there is a dependency missing.

**Comment by UZerbinati** [comment]
Did you install it with `-DUSE_OCC=OFF` instead of the default `-DUSE_OCC=ON` ?

**Comment by thelfer** [comment]
must check

**Comment by UZerbinati** [comment]
Because `unit_square` has lately become an OCC object in solve hence, if you install without OCC support it might not be there.
You should be able to test the code even without OCC if you use `from netgen.geom2d import unit_square`.

**Comment by thelfer** [comment]
Indeed. I remember that I removed it because OCC was **not** listed as a dependency to `ngsolve`

**Comment by thelfer** [comment]
I think that you have two choices :)

- update the installation guide to include `OCC` as a mandatory dependency
- update your examples

**Comment by UZerbinati** [comment]
We have included include OCC as a mandatory dependency (we will also change the pyproject.toml in next PR).

**Comment by thelfer** [comment]
Hi @UZerbinati,

Sorry for being so slow.

I reinstalled `ngsolve`, but there is still an issue with your install instructions: I had to add this package to be able to build `ngsolve` with `OpenCascade` enabled:

~~~
$ pip install  netgen-occt-devel
~~~

As a suggestion, I would add this line in your tutorial after the call to `Draw`:

~~~
input("Press the <ENTER> key to continue...")
~~~

**Comment by UZerbinati** [comment]
Thank you for spotting this. It has been fixed in PR #65.
